packages/plans-grid-next/src/components/features-grid/plan-features-list.tsx

```diff
@@ -132,7 +134,7 @@ const PlanFeaturesList = ( {
 					isTableCell={ options?.isTableCell }
 					className="plan-features-2023-grid__table-item"
 				>
-					{ featureGroup?.getTitle() && (
+					{ ! enableCategorisedFeatures && featureGroup?.getTitle() && (
```

Comment:
I don't quite follow this change. Do you mind explaining? If a title exists, then it should be rendered - that's the idea throughout here. 

It's also quite an obscured/cryptic condition semantically - "if we don't want categorised features and have a group/category title"?

Reply:
Good catch, thanks. We don't need the additional check, `getTitle` will handle this - I've updated the feature group title to be `null`. 

[5e16d3a](https://github.com/Automattic/wp-calypso/pull/94044/commits/5e16d3a2f63a2e3caf0c64e6e6c117bfa261083e)